src/consensus/aft/raft.h

```diff
@@ -166,7 +166,7 @@ namespace aft
 
     size_t entry_size_not_limited = 0;
     size_t entry_count = 0;
-    Index entries_batch_size = 1;
+    Index entries_batch_size = 20;
```

Comment:
This is another drive-by mitigation for #1451. Start with larger batches than 1, so that we have fewer AEs and faster catchup in normal operation.

Reply:
Possibly good to split in its own PR if we thought we wanted to measure the perf impact independently.